### PR TITLE
TS-4699: Fix static analysis warnings

### PR DESCRIPTION
**Pull request by PSUdaemon**

### CODE REVIEW

File: lib/atscppapi/examples/websocket/WebSocket.h [hunk 1]
```diff
@@ -38,6 +38,11 @@ using atscppapi::InterceptPlugin;
 using atscppapi::Transaction;
 using atscppapi::GlobalPlugin;
 
+namespace
+{
+GlobalPlugin *plugin;
+}
```
Comment (jpeach) [inline lib/atscppapi/examples/websocket/WebSocket.h]: Shouldn't this be in the ``.cc`` file?

**Comment by atsci** [comment]
Linux build *successful*! See https://ci.trafficserver.apache.org/job/Github-Linux/783/ for details.

**Comment by atsci** [comment]
FreeBSD build *successful*! See https://ci.trafficserver.apache.org/job/Github-FreeBSD/888/ for details.

File: lib/atscppapi/examples/websocket/WebSocket.h (hunk 1 above)
Comment (PSUdaemon) [inline lib/atscppapi/examples/websocket/WebSocket.h]: That header file is only included for by this one plugin. I can move it though. I'll also fix the coverity issues that just came out.

**Comment by atsci** [comment]
FreeBSD build *successful*! See https://ci.trafficserver.apache.org/job/Github-FreeBSD/889/ for details.

**Comment by atsci** [comment]
Linux build *successful*! See https://ci.trafficserver.apache.org/job/Github-Linux/784/ for details.

**Pull request merged**